packages/nc-gui/components/smartsheet/column/EditOrAdd.vue

```diff
@@ -84,13 +84,16 @@ const showDeprecated = ref(false)
 
 const uiTypesOptions = computed<typeof uiTypes>(() => {
   return [
-    ...uiTypes.filter(
-      (t) => geoDataToggleCondition(t) && (!isEdit.value || !t.virtual) && (!t.deprecated || showDeprecated.value),
-    ),
+    ...uiTypes
+      .filter((t) => geoDataToggleCondition(t) && (!isEdit.value || !t.virtual) && (!t.deprecated || showDeprecated.value))
```

Comment:
would suggest not to use `t` here. it's look a bit confusing with `t` from translation.